Gemfile

```diff
@@ -3,18 +3,18 @@
 source 'https://rubygems.org'
 
 ruby '2.6.6'
-gem 'rails', '~> 6.1.3'
-gem 'jbuilder', '~> 2.10'
```

Comment:
Make sure not reorder this file, especially things related to rails. Order matters in `Gemfile`s. There's a definitely formatting standards to implement in this file but reordering the imports is not one of them.